src/nameProvider.js

```diff
@@ -1,189 +1,18 @@
-	{ code: 'pa', label: t('privacy', 'Panama') },
-	{ code: 'pe', label: t('privacy', 'Peru') },
-	{ code: 'pg', label: t('privacy', 'Papua New Guinea') },
-	{ code: 'ph', label: t('privacy', 'Philippines') },
-	{ code: 'pk', label: t('privacy', 'Pakistan') },
-	{ code: 'pl', label: t('privacy', 'Poland') },
-	{ code: 'pr', label: t('privacy', 'Puerto Rico') },
-	{ code: 'pt', label: t('privacy', 'Portugal') },
-	{ code: 'py', label: t('privacy', 'Paraguay') },
-	{ code: 'qa', label: t('privacy', 'Qatar') },
-	{ code: 'ro', label: t('privacy', 'Romania') },
-	{ code: 'rs', label: t('privacy', 'Serbia') },
-	{ code: 'ru', label: t('privacy', 'Russia') },
-	{ code: 'rw', label: t('privacy', 'Rwanda') },
-	{ code: 'sa', label: t('privacy', 'Saudi Arabia') },
-	{ code: 'sb', label: t('privacy', 'Solomon Islands') },
-	{ code: 'sd', label: t('privacy', 'Sudan') },
-	{ code: 'se', label: t('privacy', 'Sweden') },
-	{ code: 'sg', label: t('privacy', 'Singapore') },
-	{ code: 'si', label: t('privacy', 'Slovenia') },
-	{ code: 'sk', label: t('privacy', 'Slovakia') },
-	{ code: 'sl', label: t('privacy', 'Sierra Leone') },
-	{ code: 'sn', label: t('privacy', 'Senegal') },
-	{ code: 'so', label: t('privacy', 'Somalia') },
-	{ code: 'sr', label: t('privacy', 'Suriname') },
-	{ code: 'ss', label: t('privacy', 'South Sudan') },
-	{ code: 'sv', label: t('privacy', 'El Salvador') },
-	{ code: 'sy', label: t('privacy', 'Syria') },
-	{ code: 'sz', label: t('privacy', 'Swaziland') },
-	{ code: 'td', label: t('privacy', 'Chad') },
-	{ code: 'tf', label: t('privacy', 'Fr. S. Antarctic Lands') },
-	{ code: 'tg', label: t('privacy', 'Togo') },
-	{ code: 'th', label: t('privacy', 'Thailand') },
-	{ code: 'tj', label: t('privacy', 'Tajikistan') },
-	{ code: 'tl', label: t('privacy', 'Timor-Leste') },
-	{ code: 'tm', label: t('privacy', 'Turkmenistan') },
-	{ code: 'tn', label: t('privacy', 'Tunisia') },
-	{ code: 'tr', label: t('privacy', 'Turkey') },
-	{ code: 'tt', label: t('privacy', 'Trinidad and Tobago') },
-	{ code: 'tw', label: t('privacy', 'Taiwan') },
-	{ code: 'tz', label: t('privacy', 'Tanzania') },
-	{ code: 'ua', label: t('privacy', 'Ukraine') },
-	{ code: 'ug', label: t('privacy', 'Uganda') },
-	{ code: 'us', label: t('privacy', 'United States') },
-	{ code: 'uy', label: t('privacy', 'Uruguay') },
-	{ code: 'uz', label: t('privacy', 'Uzbekistan') },
-	{ code: 've', label: t('privacy', 'Venezuela') },
-	{ code: 'vn', label: t('privacy', 'Vietnam') },
-	{ code: 'vu', label: t('privacy', 'Vanuatu') },
-	{ code: 'ye', label: t('privacy', 'Yemen') },
-	{ code: 'za', label: t('privacy', 'South Africa') },
-	{ code: 'zm', label: t('privacy', 'Zambia') },
-	{ code: 'zw', label: t('privacy', 'Zimbabwe') }
-]
+import { getData, getName } from 'country-list'
 
 export function getCountryList() {
-	return list.sort((a, b) => a.label < b.label ? -1 : 1)
+	return getData().map(country => {
+		return {
+			code: country.code,
+			label: t('privacy', country.name)
```

Comment:
This is not translatable.

The script that extracts our translations needs the real string, not a variable.

Reply:
Oh :(